charts/team-ns/templates/argocd/argocd-applicationset.yaml

```diff
@@ -0,0 +1,50 @@
+{{- $v := .Values }}
+{{- if $v.apps.argocd.enabled }}
+{{- range $v.workloads }}
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: team-{{ $v.teamId }}-{{ .name }}
```

Comment:
Hi Jeho,

Bit puzzled puzzled as to how this fetches the corresponding teamId for the workload at the current index/count of the range/loop. Is $v here limited solely to the values context for the current team, or is it the global values blob?

Reply:
All values provided to the teams-ns are defined in helmfile-60. In line 164 all team values are given.

Reply:
Cheers!